drivers/gpio/Kconfig.rt1718s

```diff
@@ -0,0 +1,45 @@
+# RT1718S driver configuration options
+
+# Copyright 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_RT1718S
+	bool "RT1718S I2C-based TCPC chip with GPIOs"
+	default y
+	depends on DT_HAS_RICHTEK_RT1718S_GPIO_PORT_ENABLED
+	help
+	  Enable driver GPIO for RT1718S I2C-based TCPC chip.
+
+if GPIO_RT1718S
+
+config RT1718S_INIT_PRIORITY
```

Comment:
I have would strongly suggest making this a devicetree property. What if on a system, there were too and one had to be started before the other?